variables.tf

```diff
@@ -43,6 +43,24 @@ variable "shared_access_key_enabled" {
   default     = false
 }
 
+variable "is_hns_enabled" {
+  description = "Is Hierarchical Namespace enabled. This can be used with Azure Data Lake Storage Gen 2"
+  type        = bool
+  default     = false
+}
+
+variable "queue_encryption_key_type" {
+  description = "The encryption type of the queue service. Possible values are Service and Account"
+  type        = string
+  default     = "Service"
+}
+
+variable "table_encryption_key_type" {
+  description = "The encryption type of the table service. Possible values are Service and Account"
+  type        = string
+  default     = "Service"
+}
```

Comment:
We should consider making these variables optional with default argument being `null` or similar. When running Go test on all four examples, the argument for the variables is set to `Service`. In a scenario e.g. running Terraform Import on existing resources that does not include these values, an option to exclude them would be ideal.

Reply:
@hknutsen Tagging you here so are in the loop, and perhaps could give your opinion on how we could implement this 😄

Reply:
These values are always set. According to docs, they are set to `"Service"` by default. Makes sense to me to set them to the same default value in our module.

Reply:
So even for standard storage accounts, the values are always by default set to "Service", even if the storage account is not ment as a data lake account?

Reply:
Yes, they don't seem to have anything to do with Data Lake Storage actually, but I'm guessing Stian and co. are setting different values in some of their Data Lake Storage accounts, which is why it's included in this PR 🙂 

All tests passed as well 🎉